girder/api/v1/system.py

```diff
@@ -108,7 +108,18 @@ def setSetting(self, params):
         .errorResponse('You are not a system administrator.', 403)
         .errorResponse('Failed to set system setting.', 500))
 
-    @access.admin
+    @access.admin(scope=TokenScope.SETTINGS_READ)
```

Comment:
Just to clarify, the behavior here (and the other settings endpoints remains the same):
* A token must be for a user with >= the access level AND the correct scope to access the endpoint
* Ordinary user tokens still always have `TokenScope.USER_AUTH`
* `TokenScope.USER_AUTH` is a super-scope, so every ordinary user token satisfies this new `scope=TokenScope.SETTINGS_READ` requirement

Then at some point in the future, we'll introduce tokens which don't have `TokenScope.USER_AUTH`, but do have`TokenScope.SETTINGS_READ`, so this addition will become relevant then. However, those tokens still have to satisfy the admin user requirement, so I don't totally understand what their use case would be.

Let me know if I got this right.